assets/sass/components/thank-with-google/_googlesitekit-twg-color-radio.scss

```diff
@@ -45,7 +46,7 @@
 				}
 
 				.mdc-image-radio__content[aria-checked="true"] {
-					outline: 0.1875rem solid;
+					outline: 0.125rem solid;
```

Comment:
```suggestion
					outline: 3px solid;
```

Reply:
> Why are we using rem here when the other styles use px? 🤔

I originally used `rem` here to keep it consistent with `ImageRadio`:

https://github.com/google/site-kit-wp/blob/db0c22626212552d025da373114844a9e5f6cc3a/assets/sass/components/global/_googlesitekit-image-radio.scss#L118-L120

I have now restored it back to `0.1875rem` which translates to `3px`, but do let me know if I should change it to `3px`.

Thank you, @aaemnnosttv!

Reply:
> I have now restored it back to `0.1875rem` which translates to `3px`, but do let me know if I should change it to `3px`.

I'm not sure why we use `rem` there either but it makes sense to keep them consistent for now. Thanks 👍

Reply:
Thank you, @aaemnnosttv!

Definitely, a valid question, since it looks like we use `px` for outlines everywhere else. Regardless, that's some brilliant observation and comparison, thank you!